google/oauth2/_id_token_async.py

```diff
@@ -95,7 +95,7 @@ async def _fetch_certs(request, certs_url):
 
     data = await response.data.read()
 
-    return json.loads(json.dumps(data))
+    return json.loads(data.decode("utf-8"))
```

Comment:
This works )

and a question to  @arithmetic1728: I wonder if there is a way we can decode all the responses at the time we get them... so we don't have to remember this next time

Reply:
I think this can be achieved by adding a new `decoded_data` property in parallel to the `data` property in the `_Response` class

https://github.com/googleapis/google-auth-library-python/blob/6f49d1fa1dff62c5f2ed0cf198507568f5c4296f/google/auth/transport/requests.py#L73-L75